app/safe/internal/server/route/secret/secret.go

```diff
@@ -97,27 +93,42 @@ func Secret(cid string, w http.ResponseWriter, r *http.Request, spiffeid string)
 	}
 
 	log.DebugLn(&cid, "Secret:Upsert: ", "workloadIds:", workloadIds,
-		"namespaces:", namespaces, "backingStore:", backingStore,
```

Comment:
Removed backing store from secrets.

backing store shall be set at the engine (VSecM Safe) level; not on a per-secret basis.